changelog/1.24.0.md

```diff
@@ -0,0 +1,64 @@
+---
+title: "1.24.0"
+description: "Released on 10/21/2021"
+---
+
+### Breaking changes ❗
+
+There are no breaking changes in 1.24.0.
+
+### Features ✨
+
+- web: added ability to pull images from private Amazon ECR repositories.
+- web: added alert to notify users when disks are full.
+- web: added ability for end-users to set auto-off behavior for their
+  workspaces.
+- web: added information regarding applications used to the audit log.
+- web: updated the in-product changelog to display information for multiple
+  versions of Coder.
+- web: added ability to set the background color for all in-product banners with
+  a color picker.
+- infra: added auto-injection of certificates needed by `coderd`.
+
+### Bug fixes 🐛
+
+- web: fixed rendering issues when using dark theme.
+- web: fixed issue with inability to update a registry name or URL.
+- web: fixed issue with Coder not displaying an error when there is an issue
+  during OIDC login.
+- web: fixed issue where large outputs would sometimes cause web terminals to
+  disconnect.
+- web: fixed issue with Intercom not loading for hosted beta users.
+- web: fixed issue with RStudio not launching.
+- web: fixed issue with password max length validation being too narrow for
+  registries (password length limit for image registries has been updated to 32
+  KiB).
+- web: fixed issue with incorrect dev URL status indicators
+- web: fixed issue with dev URLs sometimes not opening.
+- web: fixed issue with the **Save Preferences** button being permanently
+  disabled.
+- web: fixed issues with rendering icons in the user interface.
+- web: fixed issue with workspace templates sometimes not updating.
+- web: fixed issue with workspaces needing to be rebuilt twice after
+  regenerating an SSH key.
+- infra: fixed issue with inability to set `ulimit` inside cached CVMs.
+- api: removed ability for site managers to create site admins through the API.
+
+### Security updates 🔐
+
+- infra: removed dependency on vulnerable `jwt-go` package.
+- infra: updated login functionality to always hash passwords on login.
```

Comment:
This line item should be updated - we already hashed passwords, and this wording makes it seem like we hadn't --> it's more that we fixed a difference in the time to response based on whether an account exists or not, which is an account enumeration timing attack. See: https://app.shortcut.com/coder/story/17567/user-enumeration-timing-attack-failed-login-attempts-should-take-a-constant-time